google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineArtifactDeploymentSourceType.java

```diff
@@ -61,14 +58,9 @@ public AppEngineArtifactDeploymentSource load(@NotNull Element tag, @NotNull Pro
 
     if (settings != null) {
       Artifact[] artifacts = ArtifactManager.getInstance(project).getArtifacts();
-      Optional<Artifact> artifact = Iterables.tryFind(Arrays.asList(artifacts),
-          new Predicate<Artifact>() {
-            @Override
-            public boolean apply(Artifact artifact) {
-              return artifact.getName().equals(artifactName);
-            }
-          }
-      );
+      Optional<Artifact> artifact = Arrays.asList(artifacts).stream()
+          .filter(candidate -> candidate.getName().equals(artifactName))
```

Comment:
nitpicking here, but I like naming the parameters to match what they are; e.g. `artifact` in this case, so its easier to read.